lib/simplecov_linter_formatter/formatters/text_lines_formatter.rb

```diff
@@ -0,0 +1,31 @@
+module SimpleCovLinterFormatter
+  class TextLinesFormatter
+    def initialize(command_name, lines)
+      @command_name = command_name
+      @lines = lines
+    end
+
+    def format
+      {
+        @command_name.to_sym => {
+          coverage: group_lines_by_file
+        }
+      }
+    end
+
+    private
+
+    def group_lines_by_file
+      result = {}
+
+      @lines.each do |line|
+        file_name, line_number, column, status_lines_count = line.split(":")
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Lint/UselessAssignment: Useless assignment to variable - column. Use _ or _column as a variable name to indicate that it won't be used.